notebooks/official/matching_engine/similar_products_recommender_using_matching_engine.ipynb

```diff
@@ -0,0 +1,1683 @@
+{
```

Comment:
<p>combine 2 text cells</p><p> </p><p>Deploy Indexes -&gt; Deploy the Indexes</p><p> </p><p>Deploy ANN Index -&gt; Deploy the ANN Index</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Reply:
<p>done</p>